src/platform-apps/channels/channel-view-container.tsx

```diff
@@ -35,17 +39,20 @@ export class Container extends React.Component<Properties, State> {
     const channel = denormalize(props.channelId, state) || null;
     const {
       authentication: { user },
+      users: { users },
     } = state;
 
     return {
       channel,
       user,
+      users,
```

Comment:
we shouldn't just be grabbing all users here. .. i think a couple of steps from my initial feedback might have gotten lost in the shuffle:
b) let's trigger the fetch using an action on the channel, not the user.
c) let's add a relation to channels so that the user's are connected to the channel that they're fetched for.
d) let's access the users using the channel, as that will speed things up when re-accessing a channel, and will also make it more clear what it is we're doing. eg. myChannel.users

essentially, at this point, since we've already denormalized the channel, you should *already* have the users. accessing them should be the same as for messages: `channel.users` or `channel.members`

Reply:
https://github.com/zer0-os/zOS/blob/development/src/store/channels/index.ts
you'll need to add a relationship to the channel for members/users/whatever we want to call it, and it should be connected to users in the same way that channels are connected to messages.